src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -867,6 +867,31 @@ public enum VerticalLineAlignment
         Bottom
     }
 
+    /// <summary>
+    /// An enum of ellipsis location.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum EllipsisLocation
+    {
+        /// <summary>
+        /// ellipsis location at end.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
Please add this. [EditorBrowsable(EditorBrowsableState.Never)]

Reply:
Done